Mathlib/MeasureTheory/Group/Measure.lean

```diff
@@ -568,6 +568,24 @@ instance Measure.Regular.inv [ContinuousInv G] [Regular μ] : Regular μ.inv :=
 instance Measure.InnerRegular.inv [ContinuousInv G] [InnerRegular μ] : InnerRegular μ.inv :=
   InnerRegular.map (Homeomorph.inv G)
 
+/-- The image of an inner regular measure under map of a left action is again inner regular. -/
+@[to_additive
+   "The image of a inner regular measure under map of a left additive action is again
+    inner regular"]
+instance innerRegular_map_smul {α} [Group α] [MulAction α G] [ContinuousConstSMul α G]
```

Comment:
Does this actually need `α` to be a group? I would expect a monoid is sufficient.

Reply:
Changed to monoid @eric-wieser